utils/windowfunction/wf_stats.cpp

```diff
@@ -225,9 +224,17 @@ void WF_stats<T>::operator()(int64_t b, int64_t e, int64_t c)
     {
         setValue(CalpontSystemCatalog::DOUBLE, b, e, c, (double*) NULL);
     }
-    else if (fCount == 1 && (fFunctionId == WF__STDDEV_SAMP || fFunctionId == WF__VAR_SAMP))
+    else if (fCount == 1)
     {
-        setValue(CalpontSystemCatalog::DOUBLE, b, e, c, (double*) NULL);
+        if (fFunctionId == WF__STDDEV_SAMP || fFunctionId == WF__VAR_SAMP)
```

Comment:
I believe we are fixing something that is outside of the scope of 4188 here or am I wrong?

Reply:
This change is fixing a regression in working_tpch1_compareLogOnly/windowFunctions/MCOL-3423.sql. I did not see this test failing in develop prior to the MCOL-641 merge. So it's possible some change done to this file as part of MCOL-641 introduced this regression. Now, we are handling `fCount == 0` and `fCount == 1` here as it is handled in the non-window function aggregate versions of these functions in `RowAggregationUM::calculateStatisticsFunctions`